File: Patterns/4. Right Triangle Patterns (Top-Right)/33. Right Triangle TR Increasing Numbers/trin.java
import java.util.*;
public class trin{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a Number : ");
        int n = sc.nextInt();

        for(int i=1; i<=n; i++){
            // print spaces
            for(int j=1; j<=n-i; j++){
                System.out.print("  ");
            }
            // print stars
            for(int j=1; j<=i; j++){
                System.out.print(j + " ");
            }
            System.out.println();
        }
        sc.close();
    }
}

/* 

Output

        1 
      1 2 
    1 2 3 
  1 2 3 4 
1 2 3 4 5   

*/

File: Patterns/4. Right Triangle Patterns (Top-Right)/33. Right Triangle TR Increasing Numbers/trin.cpp
#include <iostream>
using namespace std;
int main(){
    int n;
    cout<<"Enter a Number : ";
    cin>>n;

    for(int i=1; i<=n; i++){
        // print spaces
        for(int j=1; j<=n-i; j++){
            cout<< "  ";
        }
        // print stars
        for(int j=1; j<=i;j++){
            cout<<j<<" ";
        }
        cout << '\n';
    }
}

/* 

Output

        1 
      1 2 
    1 2 3 
  1 2 3 4 
1 2 3 4 5 

*/ 